clients/identity/authentication_test.go

```diff
@@ -261,6 +261,22 @@ func TestHandler_bothTokens(t *testing.T) {
 	})
 }
 
+func TestSplitTokens(t *testing.T) {
+	Convey("Given a florence token and service token", t, func() {
+		florenceToken := "Bearer 987654321"
```

Comment:
Not that it matters for the test - but to prevent confusion it may be worth swapping the values so the service token is prefixed with Bearer, instead of the florence token.